producers/http/http.go

```diff
@@ -126,6 +126,10 @@ func (p *producerImpl) writeObjectToStore(d producers.Datapoint, m producers.Met
 	}
 	// e.g. dcos.metrics.app.kafka.server.ReplicaFetcherManager.MaxLag
 	qualifiedName := prefix + producers.MetricNamespaceSep + d.Name
+	for k, v := range d.Tags {
+		// e.g. dcos.metrics.node.network.out.errors.interface.eth0
+		qualifiedName = qualifiedName + producers.MetricNamespaceSep + k + producers.MetricNamespaceSep + v
```

Comment:
I'm not sure how I feel about this.  Adding the tag name and value to the metric name opens us up to other metric name collisions.  This can happen when another metric name (without tags) has the same qualified name as another metric after the tags are added to the qualified metric name.

For example `metric=foo.bar.baz` would collide with `metric=foo,tags={bar:baz}`

Especially since we don't control all of the metrics (app metrics) it seems that this will come back to bite us later.  I think it's likely that if we want to do this the right way we will need another approach.